app/forms/RadioButtonAmountForm.scala

```diff
@@ -71,6 +71,36 @@ object RadioButtonAmountForm {
     }
   }
 
+  def amountFormatterWithGreaterThanZero(
```

Comment:
Are there ways you could reduce the duplication with `amountFormatter`?

Reply:
This is now fixed. amountFormatter takes care of both conditions

Reply:
Good. Confirmed.